### ISSUE TITLE
error propagation in xHistogram __add__ and _sub__ are wrong

### ISSUE THREAD

**Issue by albertomanfreda** [lucabaldini/ixpeobssim#656]
```
hist.set_content(self.content + other.content,
                         self.entries + other.entries,
                         self.sumw2 + other.sumw2)
```

should be

```
hist.set_content(self.content + other.content,
                         self.entries + other.entries,
                         numpy.sqrt(self.sumw2 + other.sumw2)
```
instead.

**Comment by lucabaldini** [comment]
Thanks Alberto--do you mind making a pull request (possibly, with a unit test) so that we can get this fixed quickly?

**Comment by albertomanfreda** [comment]
tentative fix https://github.com/lucabaldini/ixpeobssim/pull/659

**Comment by lucabaldini** [comment]
Fix merged into the main branch.

**Comment by lucabaldini** [comment]
Shipped in 29.4.0
